web/vue/src/components/global/mixins/dataset.js

```diff
@@ -21,13 +21,16 @@ var mixin = {
 
         response.datasets.forEach(market => {
           market.ranges.forEach(range => {
+            let num = 1;
```

Comment:
You don't have to manually count. The forEach function provides a second argument called index which will increment once per iteration.